rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,257 @@
+          description: Format of the debug information.
+          example: IMAGE_DEBUG_TYPE_POGO
+
+        - name: type
+          level: extended
+          type: keyword
+          description: Information type generated by the debug options.
+          example: 13
+
+        - name: timestamp
+          level: extended
+          type: date
+          description: Timestamp of the debug information.
+          example: "2020-11-05T17:25:47.000Z"
+
+    - name: imports
+      level: extended
+      type: flattened
+      description: List of all imported functions
+      example: { "library_name" : "mscoree.dll", "imported_functions" : "GetFileVersionInfoSizeA" }
+
+    - name: sections
+      level: extended
+      description: >
+        Data about sections of compiled binary PE
+      type: nested
+      fields:
+        - name: chi2
+          level: extended
+          description: Chi-square probability distribution.
+          type: long
+          example: 3027194
+
+        - name: virtual_address
+          level: extended
+          description: Virtual address available to the file.
+          type: long
+          format: bytes
+          example: 8192
+
+        - name: entropy
+          level: extended
+          description: Measurement of entropy randomness in the file.
+          type: float
+          example: 6.24
+
+        - name: flags
+          level: extended
+          description: Section flags of the file.
+          type: keyword
+          example: rx
+
+        - name: name
+          level: extended
+          description: Section names of the file.
+          type: keyword
+          example: .text, .data
+
+        - name: raw_size
+          level: extended
+          description: Size of the section or the dize of the initialized data on disk.
+          type: long
+          format: bytes
+          example: 198144
+
+    - name: resources
+      level: extended
+      type: nested
+      description: >
+        If the PE contains resources, some info about them
+      fields:
+        - name: chi2
+          level: extended
+          description: Chi-square probability distribution.
+          type: long
+          example: -1
+
+        - name: filetype
+          level: extended
+          description: File type of the resources section.
+          type: keyword
+          example: Data
+
+        - name: entropy
+          level: extended
+          description: Measurement of entropy randomness in the resources section.
+          type: long
+          example: 0, 1
+
+        - name: sha256
+          level: extended
+          description: SHA256 hash of resources section.
+          type: keyword
+          example: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+        - name: language
+          level: extended
+          description: Language identification.
+          type: keyword
+          example: "CHINESE SIMPLIFIED"
+
+        - name: types
```

Comment:
This will only be a single type, see the structure here: https://docs.microsoft.com/en-us/windows/win32/menurc/resourceheader?redirectedfrom=MSDN

There should only be a single language id and type associated with a single resource ([here's](https://docs.microsoft.com/en-us/windows/win32/menurc/resource-types) the list of pre-defined resource types)